pkg/master/import_known_versions_test.go

```diff
@@ -67,6 +67,63 @@ func TestTypeTags(t *testing.T) {
 	}
 }
 
+func TestRegisteredObjectFieldSelector(t *testing.T) {
+	namespaceScopedObjects := map[string][]string{
```

Comment:
instead of hard-coding these, can you loop over api.Scheme.AllKnownTypes() and skip internal versions (see other tests in this file for an example)